### PR TITLE
fix(travis): increase wait time for pods

### PR DESCRIPTION
**Pull request by shubham14bajpai**
Signed-off-by: shubham <shubham.bajpai@mayadata.io>

**What this PR does**:
With travis-ci.org facing some infra issues, the tests may fail due to a slow VM or network. This PR increases the timeout for pods to come into running state. 

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

**Checklist**
- [ ] PR messages has document related information
- [ ] Labelled this PR & related issue with `documentation` tag
- [ ] PR messages has breaking changes related information
- [ ] PR messages has upgrade related information
- [ ] Labelled this PR & related issue with `requires-upgrade` tag
- [ ] Tests updated

### CODE REVIEW

File: ci/upgrade/setup.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ sleep 100
 
 echo "Wait for cspc-operator to start"
 
-kubectl wait --for=condition=available --timeout=300s deployment/cspc-operator -n openebs
+kubectl wait --for=condition=available --timeout=550s deployment/cspc-operator -n openebs
```
Comment (kmova) [inline ci/upgrade/setup.sh:31]: n_: update the PR on why this change was required..

Comment (shubham14bajpai) [reply ci/upgrade/setup.sh:31]: Done

**Review by kmova — APPROVED** [review]
